serverless.yml

```diff
@@ -0,0 +1,120 @@
+service: release-manager
```

Comment:
@wholesomedev I think we should decide on the name now because changing it later would not be very straightforward considering the new CF distro and Lambda associations are now managed by Serverless.

Are you okay with `route-request`?